### PR TITLE
UI libraries in fronted added

### LINKED ISSUE #1927: [ADD] <FRONTEND/UIcomponents>
### Category Name

Frontend

### Subcategory Name

UI Libraries

### Additional Context

Links for different websites which provides pre built UIcomponents.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [x] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by hritikbhattacharya**
Closes #1927 

## Changes proposed

added a sub category in Frontend section  named UI Libraries.

### CODE REVIEW

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@hritikbhattacharya 

Please add a description for UI Libraries as a subcategory.

image

**Comment by hritikbhattacharya** [comment]
ok

**Comment by Anmol-Baranwal** [comment]
@hritikbhattacharya 
You don't need to rush or anything.

Just do it whenever you have time, we will just review it after you do the changes.

**Comment by hritikbhattacharya** [comment]
@Anmol-Baranwal 
where is which folder the description of subcategory needed to be added, I didn't able to find it. would please hlep.

**Comment by Anmol-Baranwal** [comment]
@hritikbhattacharya 
You can find it here: https://github.com/rupali-codes/LinksHub/blob/main/components/TopBar/CategoryDescriptions.ts

`TopBar/CategoryDescriptions.ts`

**Comment by hritikbhattacharya** [comment]
thanks for the help.

File: database/frontend/ui-libraries.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "name": "MaterialUI",
+        "url": "https://material-ui.com/",
+        "description": "React components for faster and easier web development. Build your own design system, or start with Material Design.",
+        "category": "frontend",
+        "subcategory": "ui-libraries"
+    },
+    {
+        "name": "ChakraUI",
+        "url": "https://chakra-ui.com/",
+        "description": "Simple, Modular and Accessible UI Components for your React Applications. Built with Styled System",
```
Comment (CBID2) [inline database/frontend/ui-libraries.json:12]:
```suggestion
        "description": "Simple, Modular, and Accessible UI Components for your React Applications. Built with Styled System",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @hritikbhattacharya! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

File: database/frontend/ui-libraries.json (hunk 1 above)
Comment (hritikbhattacharya) [reply database/frontend/ui-libraries.json:12]: can you  please tell me , what changes I need to do.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

> @CBID2 
Please review this again. I think he updated the descriptions.

**Review by CBID2 — APPROVED** [review]
Well done @hritikbhattacharya! @rupali-codes, it's your turn! 😊

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**